go/wsl/hub/hub.go

```diff
@@ -141,6 +143,10 @@ func (h *Hub) newSessionFromCaps(ctx context.Context, caps *capabilities.Capabil
 		return "", nil, err
 	}
 
+	if err := maybeCreateNetLogDir(caps); err != nil {
```

Comment:
It doesn't seem WSL got into the business of creating a folder for a specific browser before. I'd like us to think twice if it's really need to introduce such a pattern now.

I think previously we rely on the environment setup process to create necessary folders for the browser and browser driver, so that WSL itself doesn't need to get into this business.

Normally, the environment setup process would have better idea about how to create and clean up the needed folders. WSL doesn't know how many web tests would be running together in the environment, so it might be safe to just keep creating netlog dir without cleaning them up or so.